docs/advanced/addons/managed-dhcp.md

```diff
@@ -0,0 +1,252 @@
+sidebar_label: Managed DHCP
+title: "Managed DHCP"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.3/advanced/managed-dhcp"/>
+</head>
+
+_Available as of v1.3.0_
+
+Beginning with v1.3.0, you can configure IP pool information and serve IP addresses to VMs running on Harvester clusters using the embedded Managed DHCP feature. This feature, which is an alternative to the standalone DHCP server, leverages the vm-dhcp-controller add-on to simplify guest cluster deployment.
+
+:::note
+Harvester uses the planned infrastructure network so you must ensure that network connectivity is available and plan the IP pools in advance.
+:::
+
+## Install and Enable the vm-dhcp-controller Add-On
+
+The vm-dhcp-controller add-on is not packed into the Harvester ISO, but you can download it from the [expreimental-addons repository](https://github.com/harvester/experimental-addons). You can install the add-on by running the following command:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/harvester/experimental-addons/main/harvester-vm-dhcp-controller/harvester-vm-dhcp-controller.yaml
+```
+
+After installation, enable the add-on on the **Dashboard** screen of the Harvester UI or using the command-line tool kubectl.
+
+![](/img/v1.3/vm-dhcp-controller/enable-addon.png)
+
+## Usage
+
+1. On the **Dashboard** screen of the Harvester UI, [create a VM Network](../../networking/harvester-network.md#create-a-vm-network).  
+
+    ![](/img/v1.3/vm-dhcp-controller/vm-network.png)
+
+1. Create an IPPool object using the command-line tool kubectl.
+
+    ```shell
+    cat <<EOF | kubectl apply -f -
+    apiVersion: network.harvesterhci.io/v1alpha1
+    kind: IPPool
+    metadata:
+      name: net-48
+      namespace: default
+    spec:
+      ipv4Config:
+        serverIP: 192.168.48.77
+        cidr: 192.168.48.0/24
+        pool:
+          start: 192.168.48.81
+          end: 192.168.48.90
+          exclude:
+          - 192.168.48.81
+          - 192.168.48.90
+        router: 192.168.48.1
+        dns:
+        - 1.1.1.1
+        leaseTime: 300
+      networkName: default/net-48
+    EOF
+    ```
+
+1. [Create a VM](../../vm/create-vm.md#how-to-create-a-vm) that is connected to the VM Network you previously created.
+
+    ![](/img/v1.3/vm-dhcp-controller/create-vm.png)
+
+1. Wait for the corresponding VirtualMachineNetworkConfig object to be created and for the MAC address of the VM's network interface to be applied to the object.
+1. Check the `.status` field of the IPPool and VirtualMachineNetworkConfig objects, and verify that the IP address is allocated and assigned to the MAC address.
+
+    ```shell
+    $ kubectl get ippools.network net-48 -o yaml                      
+    apiVersion: network.harvesterhci.io/v1alpha1
+    kind: IPPool
+    metadata:
+      creationTimestamp: "2024-02-15T13:17:21Z"
+      finalizers:
+      - wrangler.cattle.io/vm-dhcp-ippool-controller
+      generation: 1
+      name: net-48
```

Comment:
should we clarify here that `name` of `IPPool` Object needs to match the `VM Network Name` / `NetAttachDefinition'?

Reply:
ignore that.